scala-package/core/src/main/scala/org/apache/mxnet/IO.scala

```diff
@@ -216,6 +244,30 @@ object DataBatch {
       this
     }
 
+    /**
+      * Set the dtype.
+      * @param dataDType The dtype of the data, default is Float32
+      * @param labelDType The dtype of the label, default is Int32
+      * @return this
+      */
+    def setDType(dataDType: DType, labelDType: DType): Builder = {
```

Comment:
suggest to have `setDataDType` and `setLabelDType` separately, which will be helpful for Java users who want to set dtype for only one of them.